object/message.go

```diff
@@ -269,6 +269,41 @@ func (message *Message) GetId() string {
 	return fmt.Sprintf("%s/%s", message.Owner, message.Name)
 }
 
+func DeleteEmptyTextMessagesInChat(chat string) (int64, error) {
+	// find empty text messages created by user in chat
```

Comment:
You should use the history:

![image](https://github.com/user-attachments/assets/cf2544e6-35e1-4d70-b1cf-ea47ca22cb87)